src/applications/simple-forms/21-4138/config/constants.js

```diff
@@ -1,4 +1,32 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { unifyPPFormData } from '../helpers';
+
+export const PrimaryActionLink = ({ href = '/', children, onClick = null }) => (
```

Comment:
does this look okay on mobile too ? (can test in devtools with mobile option)

Reply:
Yup, looks good 👍